docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -494,7 +494,6 @@ table ``test_table`` by using the following query::
      key                   | value    |
     -----------------------+----------+
     write.format.default   | PARQUET  |
-    format-version         | 2        |
```

Comment:
That table is based on `org.apache.iceberg.Table#properties` so the contents of the table aren't very strict. There could be a key/value entry for anything in this class, if the table was made or altered in Spark https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/TableProperties.java

I"m not sure how to make that clear in the docs though

Reply:
The example should include properties a user is most likely to see.
`format-version` doesn't seem to be one of them, so it's a good change to remove it from here.

let's consider this sufficient for this PR

Reply:
In iceberg's metadata file, `format-version` is an element at the same level as properties and will not be included in properties. A sample metadata file is as follows：
`{
  "format-version" : 1,
  "location" : "hdfs://testOnline/data/iceberg/warehouse/test.db/test_table_v1",
  "properties" : {
    "commit.retry.max-wait-ms" : "3000",
    "max.compact.file.size" : "4194304",
    "commit.retry.total-timeout-ms" : "100000",
    "preserve.snapshot.days" : "3",
    "preserve.snapshot.nums" : "10",
    "write.format.default" : "parquet",
    "max.rewrite.files" : "3000",
    "write.metadata.delete-after-commit.enabled" : "true",
    "commit.retry.num-retries" : "2000",
    "commit.retry.min-wait-ms" : "100"
  },
  "current-snapshot-id" : 1234567890123,
  "...": "..."
}`